src/main/java/Application.java

```diff
@@ -0,0 +1,103 @@
+import core.basesyntax.dao.animal.AnimalDao;
+import core.basesyntax.dao.animal.AnimalDaoImpl;
+import core.basesyntax.dao.figure.FigureDao;
+import core.basesyntax.dao.figure.FigureDaoImpl;
+import core.basesyntax.dao.machine.MachineDao;
+import core.basesyntax.dao.machine.MachineDaoImpl;
+import core.basesyntax.model.figure.Circle;
+import core.basesyntax.model.figure.Triangle;
+import core.basesyntax.model.machine.Car;
+import core.basesyntax.model.machine.Machine;
+import core.basesyntax.model.machine.Truck;
+import core.basesyntax.model.zoo.Animal;
+import core.basesyntax.model.zoo.Cat;
+import core.basesyntax.model.zoo.Dog;
+import core.basesyntax.util.HibernateUtil;
+import java.util.List;
+
+public class Application {
```

Comment:
it seems to me Dmytro said not to push Application/main

Reply:
done